scripts/generate-manifest.js

```diff
@@ -94,8 +94,9 @@ const manifest = {
   content_scripts: [
```

Comment:
This is the MetaMask config which adds some more restrictions on where to inject the tag.  

![Screenshot 2023-07-20 at 15 03 07](https://github.com/hirosystems/wallet/assets/2938440/a71e79ee-7cf4-4c46-8afb-f3a264ca435f)

I have added the same `run_at` value they use also, I'm not sure if we want to add that too? The docs say the preferred and default is `document_idle`

Reply:
Maybe a better fix is to just remove support for this? 

It seems to be a request from just one person, so I'm not sure if we really need it still:
https://github.com/hirosystems/wallet/discussions/2263

Reply:
IIRC this does then also impact tools like stackblitz & codesandbox

Reply:
I spoke to Kyran about this yesterday and I will do a bit more research into this to see if we can gain more benefits from doing this optimally. 

The easiest solution for now is removing that flag but maybe other developers would like it in the future. Also if Metamask have set it it's probably for a good reason.